rasa/core/tracker_store.py

```diff
@@ -1238,7 +1421,7 @@ def _create_from_endpoint_config(
 
     if endpoint_config is None or endpoint_config.type is None:
         # default tracker store if no type is set
-        tracker_store = InMemoryTrackerStore(domain, event_broker)
+        tracker_store = AwaitableTrackerStore.create(InMemoryTrackerStore(domain, event_broker))
```

Comment:
how about we swap your approach and only wrap the `tracker_store` in an awaitable `TrackerStore` when `_load_from_module_name_in_endpoint_config` is called? We know from the inbox tracker stores that they will already support `async`